data_structures/DATA_STRUCTURES.md

```diff
@@ -1,29 +1,37 @@
-#Data Structures
+# Data Structures
+
 In this article, I will mainly introduce the most well-known data structures on a technical level but let's first start with my understanding of data structures.
 
 Different data structures are different representations of data. Due to these distinct features owned by these different representations, we could always describe a data structure from the following aspects:
 
 1. The properties of a data structure;
 
-2. Within a data structure, what kind of operations could be applied on it either to reflect some properties of a data structure or manipulate the data stored in it.
+2. The operations that could be applied on it, either to reflect some information of a data structure or manipulate the data stored in it.
+
+3. The internal representation.
```

Comment:
Thank you! ADT, that is the name. But Wiki also states the difference between ADT and data structures. If I understand correctly, this is emphasized in the added 3?